### PR TITLE
Add registry pods config to ClusterLogForwarder

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @jupierce

### CODE REVIEW

**Review by jupierce — CHANGES_REQUESTED** [review]

**Comment by deepsm007** [comment]
/test sync-rover-groups

pod timeout

**Comment by deepsm007** [comment]
Failure is related to the ci-tools postsubmit affecting the sync-rover-groups image

File: clusters/build-clusters/common_managed/rh-logging.yaml [hunk 1]
```diff
@@ -87,6 +95,12 @@ spec:
     outputRefs:
     - cloudwatch-receiver
     parse: json
+  - name: app-ci-registry-logs
+    inputRefs:
+    - image-registry-pod-logs
+    outputRefs:
+    - cloudwatch-receiver
+    parse: json
```
Comment (jupierce) [inline clusters/build-clusters/common_managed/rh-logging.yaml:103]: The registry does not log in structured JSON.

**Review by jupierce — CHANGES_REQUESTED** [review]

**Pull request merged**